src/events/interactionCreate.ts

```diff
@@ -25,6 +28,26 @@ export default async function (client: CustomClient, interaction: Interaction) {
   //   }
   // }
 
+  if (interaction.isModalSubmit()) {
+    const goodPerson = interaction.fields.getTextInputValue("goodPersonInput");
+    const soWhy = interaction.fields.getTextInputValue("soWhyInput");
```

Comment:
maybe check that this interaction was started with the slash command in question, this will immediately break in the future if not.